### DISCUSSION TITLE
F Keys on gnome

### DISCUSSION THREAD

**Discussion by Gaboom63** [ading2210/shimboot#173] [Q&A]
Hi! I was wondering how I would enable the function keys here, instead of the volume up, brightness etc.

**Comment by SniverDaBest** [comment]
GNOME Tweaks supports some rebinding features. It might support changing the function keys to the normal F keys. If that doesn't work, there's some other programs like [kanata](https://github.com/jtroo/kanata) to do the same thing as Tweaks would.
